### PR TITLE
Add Calm

### PR DESCRIPTION
**Pull request by Yang-Nankai**
Calm is a meditation and mental wellness platform that provides tools to reduce stress, improve sleep, and enhance mindfulness. It offers guided meditations, sleep stories, breathing exercises, and music tracks aimed at promoting relaxation and mental clarity. Calm is widely used for managing anxiety, improving sleep quality, and cultivating a calm mind. The platform offers 2FA for the account login verification process. The website is ranked 23,232 on [Similarweb](https://pro.similarweb.com/#/digitalsuite/websiteanalysis/overview/website-performance/*/999/1m?webSource=Total&key=calm.com). The website provides documentation on how to turn on 2FA.

### CODE REVIEW

**Comment by Carlgo11** [comment]
Did you forget to change the category @Yang-Nankai? :sweat_smile: 
Maybe you meant to use _health_?

File: entries/c/calm.com.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "Clam": {
```
Comment (hkamran80) [inline entries/c/calm.com.json]:
```suggestion
  "Calm": {
```

File: entries/c/calm.com.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    "domain": "calm.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://support.calm.com/hc/articles/27928137411995",
+    "categories": [
+      "marketing"
```
Comment (hkamran80) [inline entries/c/calm.com.json]:
```suggestion
      "health"
```

File: entries/c/calm.com.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "Clam": {
+    "domain": "calm.com",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://support.calm.com/hc/articles/27928137411995",
```
Comment (hkamran80) [inline entries/c/calm.com.json:7]: Calm has three different MFA systems for their customers, [partners](https://support.calm.com/hc/articles/27928137411995), and [health customers](https://www.calmhealth.com/mfa). I can't find their documentation for their customers, so could you please attach some screenshots of the MFA setup process, login process, or management screen. Please blur or crop out any personal or sensitive information.

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Comment by Yang-Nankai** [comment]
> Did you forget to change the category @Yang-Nankai? 😅 Maybe you meant to use _health_?

I'm so sorry. 😢 I forgot to change it, it should be `health` category! @Carlgo11

File: entries/c/calm.com.json (hunk 3 above)
Comment (Yang-Nankai) [reply entries/c/calm.com.json:7]: Of course I can, but I'm on vacation, I will give you feedback as soon as possible. ❤️

Comment (hkamran80) [reply entries/c/calm.com.json:7]: Enjoy your vacation!

**Review by fletcherdodd — DISMISSED** [review]

File: entries/c/calm.com.json (hunk 3 above)
Comment (Yang-Nankai) [reply entries/c/calm.com.json:7]: I think we can temporarily close this PR. I tested my account and found that I couldn’t log in with my phone number (but it still needs a phone number), so I want to wait until the 2FA start process is displayed after the successful login before opening this PR. What do you think? @hkamran80

**Pull request closed without merging**